### DISCUSSION TITLE
Windows 98SE does not detect mouse

### DISCUSSION THREAD

**Discussion by KendrickLamar2003** [utmapp/UTM#4079] [UTM Help (macOS)] [answered]
**Describe the issue**
I decided that I would want to install Windows 98SE on UTM because I was bored. I downloaded the retail full version from WinWorld and configured the VM using this video by jensd_be:
https://www.youtube.com/watch?v=jEANNpDpVtI&t=207s&ab_channel=jensd_be
When I booted the vm, I went through formatting and got to the GUI Installer. But for some reason, the cursor does not move. Tried capturing multiple times, did on work.
**Configuration**
* UTM Version: 3.2.4 (58)
* OS Version: 12.5 Beta (21G5027d)
* Intel or Apple Silicon? Apple M1

**Debug log**
[debug.log.zip](https://github.com/utmapp/UTM/files/8799278/debug.log.zip)

**Upload VM**
[config.plist.zip](https://github.com/utmapp/UTM/files/8799280/config.plist.zip)

**Comment by osy** [comment, marked as answer]
You need to disable USB from Input or enable PS/2 from QEMU tweaks.

**Reply by KendrickLamar2003** [reply]
That worked!

**Reply by msmuenchen** [reply]
"Force PS/2" does not work at the moment for me in setup. Tested with "Windows 98 Second Edition (OEM Full)" on UTM 4.4.5 on 2022 M2 MBA, Sonoma 14.4.1.

**Reply by KendrickLamar2003** [reply]
Nobody's gonna reply to this discussion, just create an issue

**Reply by sxunix** [reply]
I met the same issue. Anyone can help?

**Reply by KendrickLamar2003** [reply]
1. Enable "Force PS/2" in the QEMU settings
If that doesn't work;
2. Make a new issue

**Comment by sxunix** [comment]
> "Force PS/2" does not work at the moment for me in setup. Tested with "Windows 98 Second Edition (OEM Full)" on UTM 4.4.5 on 2022 M2 MBA, Sonoma 14.4.1.

I met the same issue. Anyone can help?

**Comment by ivanfitenko** [comment]
Other than enabling 'Force PS/2', you also need to click "Capture input devices" when the machine is running. That was what made it work for me.

**Comment by guilhermepissal-sketch** [comment]
https://m.youtube.com/watch?v=IbqNETRshw0&ra=m
